packages/vx-shape/src/index.js

```diff
@@ -3,9 +3,18 @@ export { default as Pie } from './shapes/Pie';
 export { default as Line } from './shapes/Line';
 export { default as LinePath } from './shapes/LinePath';
 export { default as LineRadial } from './shapes/LineRadial';
-export { default as LinkHorizontal } from './shapes/LinkHorizontal';
-export { default as LinkVertical } from './shapes/LinkVertical';
-export { default as LinkRadial } from './shapes/LinkRadial';
+export { default as LinkHorizontal } from './shapes/link/diagonal/LinkHorizontal';
+export { default as LinkVertical } from './shapes/link/diagonal/LinkVertical';
+export { default as LinkRadial } from './shapes/link/diagonal/LinkRadial';
```

Comment:
Will probably need to export `./shapes/link/diagonal/{LinkHorizontal, LinkVertical, LinkRadial}` from `./shapes/{LinkHorizontal, LinkVertical, LinkRadial}` else it's a breaking change for folks deep linking to the components

Reply:
I tried to fix this but was running into problems (I don't know if it was from Lerna or something else).

For example, I recreated `src/shapes/LinkHorizontal.js` and added `export * from './link/diagonal/LinkHorizontal';` to re-export from this location, but when I tried up update the reference in `vx-demo` `tree.js` from

```js
import { LinkHorizontal } from '@vx/shape';
```
to
```js
import LinkHorizontal from '@vx/shape/shapes/LinkHorizontal'
```
I'm getting a dependency error.  

![image](https://user-images.githubusercontent.com/177476/32743598-dce277ec-c87a-11e7-8bff-d18fa3467940.png)

I tried running `npm run build` within `vx-shape` and restarting `npm run dev` on `vx-demo.

Any ideas?

Reply:
does this work? 

```js
// src/shapes/LinkHorizontal.js
export { default as LinkHorizontal } from './shapes/link/diagonal/LinkHorizontal';
```

Reply:
It does not.

After that change, I also tried importing using

```js
import LinkHorizontal from '@vx/shape/shapes/LinkHorizontal'
```
and
```js
import LinkHorizontal from '@vx/shape/LinkHorizontal'
```

Reply:
No worries, i'll take a look after merging.